### ISSUE TITLE
The debugger is not fully operational for almost a month and a half.

### ISSUE THREAD

**Issue by mezoni** [Dart-Code/Dart-Code#1532]
If we take this issue (https://github.com/Dart-Code/Dart-Code/issues/1443) as a point of reference, it turns out that all this has not been moving from the dead center for many days (at this moment 41 days).
Is there a chance that this may take even more time or even longer?

**Comment by DanTup** [comment]
Please don't deliberately open duplicate issues.

The issue you reference is believe to be a VM bug (https://github.com/dart-lang/sdk/issues/35859). 17 days ago it was identified as being blocked on another issue (https://github.com/dart-lang/sdk/issues/36023) which was assigned out 6 days ago.

I'm sorry it's not moving as fast as we would like, but it is moving.

In the meantime, if your patch in #1499 works well enough for you, I can provide you instructions on making a build that you can install into your local VS Code to resolve the issue for you in the meantime.

**Comment by mezoni** [comment]
I asked you some questions here (https://github.com/Dart-Code/Dart-Code/pull/1499#issuecomment-472780833). Can you answer on them?
These questions relate to how you make decisions when developing your software.
It seems to me that so many users will be interested in how you make decisions?
